src/read.c

```diff
@@ -3227,7 +3227,15 @@ static avifResult avifParseSampleDescriptionBox(avifSampleTable * sampleTable,
 {
     BEGIN_STREAM(s, raw, rawLen, diag, "Box[stsd]");
 
-    AVIF_CHECKERR(avifROStreamReadAndEnforceVersion(&s, 0), AVIF_RESULT_BMFF_PARSE_FAILED);
+    uint8_t version;
+    AVIF_CHECK(avifROStreamReadVersionAndFlags(&s, &version, NULL));
```

Comment:
IMPORTANT: Change `AVIF_CHECK` to `AVIF_CHECKERR(..., AVIF_RESULT_BMFF_PARSE_FAILED)`, as in the original code.

Reply:
Done.